kubernetes/helm/kube-prometheus-stack/values.yaml

```diff
@@ -38,10 +38,13 @@ prometheus-operator:
   prometheus:
     prometheusSpec:
       enableAdminAPI: true
+      podMonitorSelectorNilUsesHelmValues: false
```

Comment:
@davidshare This is what tells prometheus to scrape the PodMonitor from the etcd chart.